Dockerfile.ccpp

```diff
@@ -0,0 +1,67 @@
+# on macOS this image requires rosetta 2 support for virtualization
```

Comment:
@WardF this is the docker file that builds the intel compilers. You'd only need the first 35 lines